### ISSUE TITLE
Project logo: Smaller versions and integration

### ISSUE THREAD

**Issue by LinusCDE** [toltec-dev/toltec#209]
This issue continues the original "Project logo" issue #194.

Goal is to use the logo we have as a template to create new smaller versions e.g for:

- toltec's GitHub organization logo
- Website favicon
- Readme title logo

For reference, here is the current (first) project logo in full size:

![img](https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png)

<details>

<summary>(Resized versions)</summary>
<img src="https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png" width="50px"><img src="https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png" width="100px"><img src="https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png" width="200px"><img src="https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png" width="300px">

</details>

**Comment by Eeems** [comment]
While the logo is great for large display, when you start getting smaller, it has too much detail. So it especially will not work in a favicon. So a simpler vector based logo that still looks good at a smaller size is idea.

**Comment by LinusCDE** [comment]
@Eeems Agreed. I should have put more emphasis on creating **new** smaller logos.

I added the resized variants only for demonstration.

**Comment by Eeems** [comment]
Fair enough :)

Also, the logos don't work well on dark backgrounds :(
![image](https://user-images.githubusercontent.com/1176979/104132538-31495f00-533b-11eb-9a34-e5db4c12b4fd.png)

**Comment by LinusCDE** [comment]
Good point. I think @raisjn simply turned white to transparent to make it easier to use.

I re added white background here.

<img src="https://user-images.githubusercontent.com/22298664/104133139-ea328c80-5381-11eb-99f0-7e200a063b51.png" width="50%">

[logo_with_bg_gimp.tar.gz](https://github.com/toltec-dev/toltec/files/5793194/logo_with_bg_gimp.tar.gz)

Should be rougly the same on white websites (used #fafafa for white) and looks good in dark gh as well imo.
